runtime/src/vm/opcode.rs

```diff
@@ -1724,8 +1724,8 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
   cpu!("copytov", |args, hand_mem, _, _| {
     // args = [arr_addr, inner_addr, outer_addr]
     // copy data from outer_addr to inner_addr in arr_addr
-    let inner = hand_mem.read_fixed(args[2]);
-    hand_mem.copy_to_fractal_mem(args[0], args[1], inner);
+    let inner = hand_mem.read_fixed(args[1]);
+    hand_mem.copy_to_fractal_mem(args[0], args[2], inner);
```

Comment:
This only worked before when we had no pointers because the only thing we ever copy is an empty string, which "looked" the same as fixed data.